### ISSUE TITLE
Internationalisation

### ISSUE THREAD

**Issue by smolBlackCat** [uowuo/abaddon#308]
It'd be cool if the app was available in other languages as well. I'm up for opening a pull request

**Comment by smolBlackCat** [comment]
![Screenshot from 2024-06-09 18-38-54](https://github.com/uowuo/abaddon/assets/66385489/e3a13884-ee2f-4425-8b66-c936333f44e2)
Brazilian Portuguese Translation

**Comment by ouwou** [comment]
i didnt keep internationalization in mind so none of the strings are marked for translation but ill see how easy it is to get it setup with glib/cmake and if that works out i can start throwing stuff into a translation file

**Comment by smolBlackCat** [comment]
> i didnt keep internationalization in mind so none of the strings are marked for translation but ill see how easy it is to get it setup with glib/cmake and if that works out i can start throwing stuff into a translation file

Yeah I noticed. But I'm doing some work on it already. If you want, I can open a draft pull request.

**Comment by ouwou** [comment]
sure ill take a look

**Comment by smolBlackCat** [comment]
Everything was fun and games until auditlogpane.cpp 💀. Still working on it though
